python-squid/src/main/java/org/sonar/python/PythonVisitorContext.java

```diff
@@ -41,20 +41,18 @@
 
 
   public PythonVisitorContext(AstNode rootAst, PyFileInputTree rootTree, PythonFile pythonFile) {
-    this(rootAst, rootTree, pythonFile, null);
+    this.rootAst = rootAst;
+    this.rootTree = (PyFileInputTreeImpl) rootTree;
```

Comment:
I don't get why the `rootTree` field is declared as `PyFileInputTreeImpl` instead of `PyFileInputTree`.

Reply:
good point, this was probably not necessary before either. Sticking to interface.